core/types/l2trace.go

```diff
@@ -0,0 +1,139 @@
+package types
+
+import (
+	"io"
+	"sort"
+	"strings"
+
+	"github.com/scroll-tech/go-ethereum/rlp"
+)
+
+// ExecutionResult groups all structured logs emitted by the EVM
+// while replaying a transaction in debug mode as well as transaction
+// execution status, the amount of gas used and the return value
+type ExecutionResult struct {
+	Gas         uint64         `json:"gas"`
+	Failed      bool           `json:"failed"`
+	ReturnValue string         `json:"returnValue"`
+	StructLogs  []StructLogRes `json:"structLogs"`
+}
+
+type rlpExecutionResult struct {
```

Comment:
Why remove the `ReturnValue` in the `rlpExecutionResult`?

Reply:
Receipt(evm execution result) don't have ReturnValue, so if we need we have to change receipt struct.

Reply:
Added.